blog-article.html

```diff
@@ -11,28 +11,28 @@
 <body>
     <header class="header">
         <div class="header__content">
-            <a href="index.html" class="logo">
+            <h1 class="logo">
                 <img src="img/logo.svg" width="188" height="44" alt="binabox" >
-            </a>
+            </h1>
             <nav class="navigation">
                 <ul class="navigation__list">
                     <li class="navigation__item active">
-                        <a href="index.html" class="navigation__link">
+                        <a href="#" class="navigation__link">
```

Comment:
Everything was right here. The home page is an index ✋

Reply:
Done